client/driver/rkt.go

```diff
@@ -133,33 +133,46 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 	envVars.ClearAllocDir()
 
 	for k, v := range envVars.Map() {
-		cmd_args = append(cmd_args, fmt.Sprintf("--set-env=%v=%v", k, v))
+		cmdArgs = append(cmdArgs, fmt.Sprintf("--set-env=%v=%v", k, v))
 	}
 
 	// Disble signature verification if the trust command was not run.
-	if !trust_cmd {
-		cmd_args = append(cmd_args, "--insecure-skip-verify")
+	if !trustCmd {
+		cmdArgs = append(cmdArgs, "--insecure-skip-verify")
 	}
 
 	// Append the run command.
-	cmd_args = append(cmd_args, "run", "--mds-register=false", img)
+	cmdArgs = append(cmdArgs, "run", "--mds-register=false", img)
 
 	// Check if the user has overriden the exec command.
-	if exec_cmd, ok := task.Config["command"]; ok {
-		cmd_args = append(cmd_args, fmt.Sprintf("--exec=%v", exec_cmd))
+	if execCmd, ok := task.Config["command"]; ok {
+		cmdArgs = append(cmdArgs, fmt.Sprintf("--exec=%v", execCmd))
 	}
 
+	if task.Resources.MemoryMB == 0 {
+		return nil, fmt.Errorf("Memory limit cannot be zero")
+	}
+	if task.Resources.CPU == 0 {
+		return nil, fmt.Errorf("CPU limit cannot be zero")
+	}
+
+	// Add memory isolator
+	cmdArgs = append(cmdArgs, fmt.Sprintf("--memory=%vM", int64(task.Resources.MemoryMB)*1024*1024))
```

Comment:
Yes, this was added in 0.14.0, https://github.com/coreos/rkt/pull/1851
I assumed we were targeting to converge after post 1.0, so that it's okay to not be backwards compatible for now. If we think the check is necessary, I can add it.